### DISCUSSION TITLE
Two error messages on illumos

### DISCUSSION THREAD

**Discussion by jgmills** [zrepl/zrepl#626] [Q&A] [answered]
I've been testing zrepl on openindiana, an illumos distro.  Here's part of my configuration:

```
jobs:
  - type: sink
    name: "local_sink"
    root_fs: "dpool/sink"
    serve:
      type: local
      listener_name: localsink

  - type: push
    name: "backup_system"
    connect:
      type: local
      listener_name: localsink
      client_identity: local_backup
    filesystems: {
      "dpool/origin": true,
    }
```

The first error contained the message "placeholder filesystem encryption handling is unspecified".  I was able to work around it by pre-creating the placeholder filesystems, using plain "zfs create" with no options except for the filesystem name.  However, I see that a better solution is to add the following to the configuration:

```
  recv:
    ...
    placeholder:
      encryption: unspecified | off | inherit
```

I don't know which value to choose.  "inherit" seems best.  Is it?

The second error contains this text "destination ... has been modified\nsince most recent snapshot\n".  This was likely caused by me looking at the files after they had been replicated.  That filesystem should have been created with the option "-o atime=off".  How do I do that in my configuration?  The only example I could find looks like this:

    properties:
      inherit:
        - "mountpoint"
      override: {
        "org.openzfs.systemd:ignore": "on"
      }
```

The override specification is meaningless to me.  How do I express the equivalent of "-o atime=off"?

**Comment by problame** [comment, marked as answer]
Sorry for the late response, @jgmills .

> I don't know which value to choose. "inherit" seems best. Is it?

It depends on your use case.
Explanation is here: https://zrepl.github.io/configuration/sendrecvoptions.html#placeholders


> The override specification is meaningless to me.  How do I express the equivalent of "-o atime=off"?

```
properties:
  override: {
    "atime": "off"
  }
```

**Comment by jgmills** [comment]
I can't predict the use case.  I chose "inherit" for an example.

By reading the man page, I finally figured out that "org.openzfs.systemd:ignore" was a user-defined ZFS property.  Again I chose "atime": "off" as an example.  This was passed by "zrepl configcheck".

PS: Openindiana now has a package called backup/zrepl that can be installed on any OI system.
